updates/3/updates.sql

```diff
@@ -70,11 +70,73 @@ INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0x5273063725a43
+UPDATE assets SET coingecko="smart-mfg", name="Smart MFG" WHERE identifier="_ceth_0x6710c63432A2De02954fc0f851db07146a6c0312";/*Update coingecko*/
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0x6710c63432A2De02954fc0f851db07146a6c0312", 18, NULL); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0x6710c63432A2De02954fc0f851db07146a6c0312", "C", "Smart MFG", "MFG", 1508284800, NULL, "smart-mfg", "MFG", "0x6710c63432A2De02954fc0f851db07146a6c0312");
+UPDATE assets SET coingecko=NULL WHERE identifier="_ceth_0xB563300A3BAc79FC09B93b6F84CE0d4465A2AC27";/*Update coingecko*/
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xB563300A3BAc79FC09B93b6F84CE0d4465A2AC27", 18, NULL); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xB563300A3BAc79FC09B93b6F84CE0d4465A2AC27", "C", "Red Cab", "REDC", 1526203977, NULL, NULL, "REDC", "0xB563300A3BAc79FC09B93b6F84CE0d4465A2AC27");
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xCC8Fa225D80b9c7D42F96e9570156c65D6cAAa25", 0, NULL); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xCC8Fa225D80b9c7D42F96e9570156c65D6cAAa25", "C", "Smooth Love Potion", "SLP", 1619421512, NULL, "smooth-love-potion", "SLP", "0xCC8Fa225D80b9c7D42F96e9570156c65D6cAAa25");
+*
+UPDATE assets SET swapped_for="_ceth_0xCC8Fa225D80b9c7D42F96e9570156c65D6cAAa25", coingecko="smooth-love-potion" WHERE identifier="_ceth_0x37236CD05b34Cc79d3715AF2383E96dd7443dCF1";/*Migrate asset*/
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0x37236CD05b34Cc79d3715AF2383E96dd7443dCF1", 18, NULL); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0x37236CD05b34Cc79d3715AF2383E96dd7443dCF1", "C", "Small Love Potion", "SLP", 1576751331, "_ceth_0xCC8Fa225D80b9c7D42F96e9570156c65D6cAAa25", "smooth-love-potion", "SLP", "0x37236CD05b34Cc79d3715AF2383E96dd7443dCF1");
+UPDATE assets SET coingecko=NULL WHERE identifier="_ceth_0xFD25676Fc2c4421778B18Ec7Ab86E7C5701DF187";/*Update coingecko*/
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xFD25676Fc2c4421778B18Ec7Ab86E7C5701DF187", 18, NULL); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xFD25676Fc2c4421778B18Ec7Ab86E7C5701DF187", "C", "Alchemy", "ACOIN", 1581922577, NULL, NULL, "ACOIN", "0xFD25676Fc2c4421778B18Ec7Ab86E7C5701DF187");
+UPDATE assets SET name="Fnk.com", coingecko="fnkcom" WHERE identifier="_ceth_0xB5FE099475d3030DDe498c3BB6F3854F762A48Ad";/*Migrate asset*/
```

Comment:
Why rename this? The name is finiko in the contract. For ethereum tokens, unless there is a valid good reason, we always take the name of the token.

Also Why does the comment say migrate asset? You are not migrating anything.